src/ExpressionParser.php

```diff
@@ -278,13 +278,9 @@ private function parseConditionalExpression($expr): AbstractExpression
                     // Ternary without else (expr ? expr2)
                     $expr3 = new ConstantExpression('', $this->parser->getCurrentToken()->getLine());
                 }
-            } else {
-                // Ternary without then (expr ?: expr3)
```

Comment:
This is now a proper operator in CoreExtension.

Reply:
You still need this `else` clause to define the variables (see the phpstan error) or to throw an exception about invalid syntax.

Reply:
I've removed the if as it's not possible anymore to have a `?` operator followed by the : `operator`.